chart/templates/mayastor/agents/core/agent-core-and-ha-cluster-deployment.yaml

```diff
@@ -57,3 +57,34 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+        - name: agent-ha-cluster
+          resources:
+            limits:
+              cpu: {{ .Values.agents.ha.cluster.resources.limits.cpu | quote }}
+              memory: {{ .Values.agents.ha.cluster.resources.limits.memory | quote }}
+            requests:
+              cpu: {{ .Values.agents.ha.cluster.resources.requests.cpu | quote }}
+              memory: {{ .Values.agents.ha.cluster.resources.requests.memory | quote }}
+          image: "{{ .Values.image.registry }}/{{ .Values.image.repo }}/mayastor-agent-ha-cluster:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          args:
+            - "-g=0.0.0.0:11500"
+            - "--store {{ .Release.Name }}-etcd:{{ .Values.etcd.service.port }}"
+            - "--core-grpc=0.0.0.0:50051"
```

Comment:
```suggestion
            "--core-grpc=https://{{ .Release.Name }}-agent-core:50051"
```

Reply:
Ahh! Right!

Reply:
Although since we're running on the same pod, maybe we can just connect directly however we'd need to add some life/ready probing to make sure we don't see crashes/failures if they start out of order?